src/coins.cpp

```diff
@@ -1923,7 +2023,7 @@ double CCoinsViewCache::GetPriority(const CTransactionBase &tx, int nHeight) con
             dResult += coins->vout[txin.prevout.n].nValue * (nHeight-coins->nHeight);
         }
     }
-
+    // Q: do we need to consider CSW inputs? set the coin height = 1?
```

Comment:
If we consider the CSW coin age equals to 0 -> CSW inputs decrease the priority of the Tx.
@abi87 what do you think about this?

Reply:
I think we should prioritize tx spending CSW coins like we do for ordinary txes. The way I understand priority is along the line of what Gavin Andresen explains here (https://www.youtube.com/watch?v=rQ3e1Pzu7iI min 3:39). As of now, I see no reason to do differently

Reply:
Let's discuss this